### ISSUE TITLE
Program broken after upgrading to v9.0.1 on Arch Linux

### ISSUE THREAD

**Issue by alexg-k** [jbarlow83/OCRmyPDF#418]
**Describe the issue**
After upgrading OCRmyPDF to v9.0.1 I get the following error message using the program exactly the same as before running:

```bash
ocrmypdf -l deu+eng --rotate-pages --deskew input.pdf output.pdf
```

**Expected behavior**
OCR my PDFs properly. Instead I get the following error message:

```/nas/ScanInbox//20180101_000828_0.pdf
Scan: 100%|████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00, 751.40page/s]
   INFO - Start processing 2 pages concurrent
   INFO - Using Tesseract OpenMP thread limit 4
OCR: 100%|██████████████████████████████████████████████████████████████████████████████████████| 2.0/2.0 [00:10<00:00,  5.18s/page]
  ERROR - An exception occurred while executing the pipeline
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 304, in _load_from
    self._xmp = parse(BytesIO(data))
  File "src/lxml/etree.pyx", line 3467, in lxml.etree.parse
  File "src/lxml/parser.pxi", line 1856, in lxml.etree._parseDocument
  File "src/lxml/parser.pxi", line 1876, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1764, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1127, in lxml.etree._BaseParser._parseDoc
  File "src/lxml/parser.pxi", line 601, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 711, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 640, in lxml.etree._raiseParseError
  File "<string>", line 21
lxml.etree.XMLSyntaxError: Opening and ending tag mismatch: none line 21 and xapMM:DocumentID, line 21, column 58

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 308, in _load_from
    self._xmp = parse(BytesIO(data))
  File "src/lxml/etree.pyx", line 3467, in lxml.etree.parse
  File "src/lxml/parser.pxi", line 1856, in lxml.etree._parseDocument
  File "src/lxml/parser.pxi", line 1876, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1764, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1127, in lxml.etree._BaseParser._parseDoc
  File "src/lxml/parser.pxi", line 601, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 711, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 640, in lxml.etree._raiseParseError
  File "<string>", line 21
lxml.etree.XMLSyntaxError: Opening and ending tag mismatch: none line 21 and xapMM:DocumentID, line 21, column 58

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ocrmypdf/_sync.py", line 338, in run_pipeline
    exec_concurrent(context)
  File "/usr/lib/python3.7/site-packages/ocrmypdf/_sync.py", line 292, in exec_concurrent
    pdf = ocrgraft.finalize()
  File "/usr/lib/python3.7/site-packages/ocrmypdf/_graft.py", line 282, in finalize
    self.pdf_base.save(self.output_file)
  File "/usr/lib/python3.7/site-packages/pikepdf/_cpphelpers.py", line 57, in update_xmp_pdfversion
    with pdf.open_metadata(set_pikepdf_as_editor=False, update_docinfo=False) as meta:
  File "/usr/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 198, in wrapper
    self._load()
  File "/usr/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 300, in _load
    self._load_from(data)
  File "/usr/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 310, in _load_from
    raise PdfError() from e
pikepdf._qpdf.PdfError
```

**System:**

I am using: 
- OS: Arch Linux x86_64 (kernel: 5.2.8-arch1-1-ARCH)
- OCRmyPDF Version: v9.0.1

**Comment by jbarlow83** [comment]
It will be easier if I can look at the actual file. If you are concerned about sharing the file publicly, you can encrypt it with my public key as described here:
https://github.com/jbarlow83/OCRmyPDF/wiki

It's also possible that something is wrong with Arch's latest build of `lxml`.

**Comment by alexg-k** [comment]
Sorry, this is very personal information I have scanned and I am not willing to share at all.

I can tell you this: I am using the same document scanner for all my documents and this error seems to be a one time thing. I scanned the same document a second time and the script worked as expected only removing some metadata.

Since I assume this error will not reoccure any time soon, closing the issue might make sense though the error was not resolved.
